package.json

```diff
@@ -10,14 +10,20 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "aos": "^2.3.4",
     "classnames": "^2.2.6",
     "framer-motion": "^2.3.0",
     "prop-types": "^15.7.2",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
+    "react-instagram-embed": "^1.5.0",
```

Comment:
I didn't see Instagram posts, so can we remove this?

Reply:
removed this